### DISCUSSION TITLE
Error HELL!!

### DISCUSSION THREAD

**Discussion by BenJarrett** [nss-evening-cohort-14/evening-client-side#145] [Look at what I learned!]
I was curious as to why my DOM was not rendering as I was wanted it to. 
When I was trying to follow along with today's class, I was not checking and fixing the errors I was getting in my terminal. 
When I finally had to stop and just follow along, I couldn't figure out what I needed to do in order to start the next step of creating authors. 

I was in ERROR HELL. I really learned that when coding along in class, I need to pay attention to the errors in my terminal and MAKE SURE to fix those before trying to code along. This way, I can at least have a starting point so that when I make a discussion ticket, the instructors will know exactly where my code is going wrong, instead of having to wait for me to get my errors fixed first and then have an idea of where my code is going wrong.

**Comment by drteresavasquez** [comment]
It was HELL for sure! 😂
